image-rs/scripts/install_offline_fs_kbc_files.sh

```diff
@@ -25,11 +25,19 @@ if [ "${1:-}" = "install" ]; then
     sudo install --owner=root --group=root --mode=0640 "${test_resource_record}" "${target_resource_record_path}"
     sudo install --owner=root --group=root --mode=0640 "${test_keys}" "${target_keys_path}"
 
+    # This is a workaround for CDH to read the aa_kbc_params from a
+    # file rather than kernel commandline. This makes a fake environment
+    # that CDH will recognize as it is in "peer pod", so aa_kbc param will
+    # be read from a file.
+    # TODO: when CDH has its own configuration launch file, we can
+    # promote this way.
+    mkdir -p /run/peerpod
+    touch /run/peerpod/daemon.json
+    echo "aa_kbc_params = \"offline_fs_kbc::null\"" > /etc/agent-config.toml
```

Comment:
Looking at this again reminds me how much I don't like the current peer pods workaround. Things are going to be a lot nicer when we have a real configuration file. This is kind of a clever workaround for now, though.

Reply:
Thanks. A configuration file will be much easier to handle `aa_kbc_params`, `kms credentials`, ... etc.